.github/workflows/dev-short-tests.yml

```
@@ -445,8 +445,20 @@ jobs:
         COPY build\VS2010\bin\${{matrix.platform}}_${{matrix.configuration}}\*.exe tests\
         CD tests
         sh -e playTests.sh
-        DIR
-        .\fuzzer.exe -T2m
+        fuzzer.exe -T2m
+
+  cygwin-tests:
+    runs-on: windows-latest
+    steps:
+    - run: git config --global core.autocrlf input
+    - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # tag=v3
+    - uses: cygwin/cygwin-install-action@master
```

We shouldn't be using `@master` here, we should use the commit of the latest tag https://github.com/cygwin/cygwin-install-action/releases/tag/v3

```suggestion
    - uses: cygwin/cygwin-install-action@f5e0f048310c425e84bc789f493a828c6dc80a25 # tag=v3
```

---

done